eskip/eskip.go

```diff
@@ -133,6 +133,10 @@ type Route struct {
 	// E.g. PathRegexp(/\/api\//)
 	PathRegexps []string
 
+	// Base weight of the route.
+	// E.g. Weight(50)
+	Weight int
```

Comment:
We'd like to make these explicit predicate fields deprecated, and use only the Predicate slice field for new predicates. Please remove this field and create a predicate spec like the ones here: https://godoc.org/github.com/zalando/skipper/predicates

Reply:
I cannot get this to work. See my last commit, especially this: https://github.com/zalando/skipper/pull/1338/commits/c79e86d3ac8aef7a1a438f53685b1ccbf7cec75a#diff-ae803bdbe486828b2ecb3c8a0b06a9d9R139-R147

Any advice on how to structure this?

Reply:
Btw, should all these predicates be extracted to separate predicates? Seem like a nice refactoring that I can work on if you give me the OK.

Reply:
Absolutely. But please open the pull request against the release/v1 branch. Thanks in advance!